main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-utils/colorstring"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-steplib/step-custom-test-results-export/testresultexport"
+	"github.com/ryanuber/go-glob"
+)
+
+func failf(format string, args ...interface{}) {
+	log.Errorf(format, args...)
+	os.Exit(1)
+}
+
+func main() {
+	var stepConf Config
+	if err := stepconf.Parse(&stepConf); err != nil {
+		failf("Config: %s", err)
```

Comment:
the error message this package returns matches with the prefix: `Issue with input:`